tests/test_content_loader.py

```diff
@@ -793,10 +793,106 @@ def test_get_previous_question_id_for_section_with_no_questions_returns_none(sel
         })
         assert section.get_previous_question_id(None) is None
 
-    def test_get_question_as_section(self):
+    def test_get_next_question_slug(self):
```

Comment:
There's a slightly newer set of tests in `tests/test_content_section.py`, I wonder if it'd be better to add these there instead.

The problem with this file is that it takes a very coarse-grained approach, testing the entire thing from creating objects from the original datastructure to the method itself, which means they tend to act as a binary test suite – if there are any changes to the format most of these tests will fail.